pkg/resources/metrics.go

```diff
@@ -362,6 +366,33 @@ func reconcilePostgresFreeStorageAlerts(ctx context.Context, client k8sclient.Cl
 	return nil
 }
 
+func reconcilePostgresFreeableMemoryAlert(ctx context.Context, client k8sclient.Client, inst *v1alpha1.RHMI, cr *crov1.Postgres) error {
+	// dont create the alert if we are using in cluster storage
+	if strings.ToLower(inst.Spec.UseClusterStorage) == "true" {
+		logrus.Info("skipping postgres free storage alert creation, useClusterStorage is true")
+		return nil
+	}
+
+	// build and reconcile postgres low freeable memory alert
+	alertName := "PostgresFreeableMemoryLow"
+	ruleName := "postgres-freeable-memory-low"
```

Comment:
I think we need to add this rule to the [functional tests](https://github.com/integr8ly/integreatly-operator/blob/master/test/common/alerts_exist.go#L406)